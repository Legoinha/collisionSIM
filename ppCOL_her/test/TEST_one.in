################ ThePEG-repository ###############
########### Herwig/Matchbox input file ###########


##################################################
## Initiation Basics
##################################################

# Collider type (proton-proton collider)
read snippets/PPCollider.in

set /Herwig/UnderlyingEvent/MPIHandler:

# Center-of-mass energy
cd /Herwig/Generators
set EventGenerator:EventHandler:LuminosityFunction:Energy 5020.0


##################################################
## Process selection
##################################################

# Dijet QCD Matrix Element
cd /Herwig/MatrixElements/
insert SubProcess:MatrixElements[0] MEQCD2to2
insert SubProcess:MatrixElements[1] MEHeavyQuark

##################################################
## Save the generator
##################################################

# HepMC Output 
insert /Herwig/Generators/EventGenerator:AnalysisHandlers[0] /Herwig/Analysis/HepMCFile
#set /Herwig/Analysis/HepMCFile:PrintEvent 1000000
set /Herwig/Analysis/HepMCFile:Format GenEvent
set /Herwig/Analysis/HepMCFile:Units GeV_mm

# Create the run script 
do /Herwig/MatrixElements/Matchbox/Factory:ProductionMode
cd /Herwig/Generators
saverun test_ONE EventGenerator