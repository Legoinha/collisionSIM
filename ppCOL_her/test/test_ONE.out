>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
>>>> Toolkit for HEP Event Generation - ThePEG 2.3.0 <<<<<<<<<<<<<<<<<<<<<<<<<
>>>> Mon Jun 17 11:00:27 2024                                             <<<<
<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<

>> Herwig 7.3.0 / ThePEG 2.3.0

==============================================================================
Statistics for event handler 'EventHandler':
                                       generated    number of    Cross-section
                                          events     attempts             (nb)
==============================================================================
Total (from attempted events): including vetoed events            0.12(4)e+06
==============================================================================
Total (from generated events):                1            1       0.1(1)e+06
Events carry unit weights.
==============================================================================
Per matrix element breakdown:
MEQCD2to2                                      1            1       0.1(1)e+06
MEHeavyQuark                                   0            0                0
==============================================================================
